bouncer/shared/create_and_delete_multiple_orders.ts

```diff
@@ -1,54 +1,97 @@
+import assert from 'assert';
+import { ApiPromise } from '@polkadot/api';
 import { Keyring } from '../polkadot/keyring';
 import { handleSubstrateError, lpMutex } from '../shared/utils';
 import { getChainflipApi, observeEvent } from './utils/substrate';
 import { limitOrder } from './limit_order';
+import { rangeOrder } from './range_order';
+import { depositLiquidity } from './deposit_liquidity';
+import { deposits } from './setup_swaps';
 
-export async function createAndDeleteMultipleOrders(numberOfOrders: number) {
+const LP: string = '//LP_3';
+
+async function countOpenOrders(
+  baseAsset: string,
+  quoteAsset: string,
+  lp: string,
+  chainflip: ApiPromise,
+) {
+  const orders = await chainflip.rpc('cf_pool_orders', baseAsset, quoteAsset, lp);
+  if (!orders) {
+    throw Error('Rpc cf_pool_orders returned undefined');
+  }
+  let openOrders = 0;
+
+  // @ts-expect-error limit_orders does not exist on type AnyJson
+  openOrders += orders?.limit_orders.asks.length || 0;
+  // @ts-expect-error limit_orders does not exist on type AnyJson
+  openOrders += orders?.limit_orders.bids.length || 0;
+  // @ts-expect-error limit_orders does not exist on type AnyJson
+  openOrders += orders?.range_orders.length || 0;
+
+  return openOrders;
+}
+
+export async function createAndDeleteMultipleOrders(numberOfLimitOrders: number) {
   console.log(`=== cancel_orders_batch test ===`);
   await using chainflip = await getChainflipApi();
 
   const keyring = new Keyring({ type: 'sr25519' });
   keyring.setSS58Format(2112);
-  const lpUri = process.env.LP_URI || '//LP_1';
+  const lpUri = process.env.LP_URI || LP;
   const lp = keyring.createFromUri(lpUri);
 
+  await Promise.all([
+    // provide liquidity to LP_3
+    depositLiquidity('Usdc', 10000, false, LP),
+    depositLiquidity('Eth', deposits.get('Eth')!, false, LP),
+    depositLiquidity('Dot', deposits.get('Dot')!, false, LP),
+    depositLiquidity('Btc', deposits.get('Btc')!, false, LP),
+    depositLiquidity('Flip', deposits.get('Flip')!, false, LP),
+    depositLiquidity('Usdt', deposits.get('Usdt')!, false, LP),
+    depositLiquidity('ArbEth', deposits.get('ArbEth')!, false, LP),
+    depositLiquidity('ArbUsdc', deposits.get('ArbUsdc')!, false, LP),
+    depositLiquidity('Sol', deposits.get('Sol')!, false, LP),
+    depositLiquidity('SolUsdc', deposits.get('SolUsdc')!, false, LP),
```

Comment:
These should all be taking `lpUri` instead of `LP` else this test breaks if someone supplies an `LP_URI`
Would also rename `LP` to `DEFAULT_LP_URI` or something.